tests/db_tests/postgres.test.js

```diff
@@ -0,0 +1,28 @@
+const db = require('../../database/postgres.js');
+
+let isErr = false;
+let queryData;
+
+const queryCallback = (err, data) => {
+  if (err) {
+    isErr = true;
+  } else if (data) {
+    queryData = data;
+  }
+}
+
+describe('getRestaurant', () => {
+  
+  it('invokes the callback with data for a successful query', () => {
+    const goodQuery = db.getRestaurant(1, queryCallback);
+    expect(queryData).toBeDefined
```

Comment:
toBeDefined should be toBeDefined()

Reply:
Your test looks like postgres must be running in order for it to pass? Better to mock the db.